generator/src/test/java/org/apache/kafka/message/VersionsTest.java

```diff
@@ -57,8 +55,8 @@ public void testRoundTrips() {
     }
 
     private void testRoundTrip(Versions versions, String string) {
-        assertEquals(string, versions.toString());
-        assertEquals(versions, Versions.parse(versions.toString(), null));
+        assertEquals(versions.toString(), string);
```

Comment:
the fist argument is still "expected value" so we don't need to switch the args. There are many similar occurrence.